cmd/createcluster.go

```diff
@@ -98,6 +98,25 @@ type clusterConfig struct {
 func newCreateClusterCmd(runFunc func(io.Writer, clusterConfig) error) *cobra.Command {
 	var conf clusterConfig
 
+	cmd := &cobra.Command{
+		Use:   "create-cluster",
+		Short: "Create a local charon cluster",
+		Long: "Create a local charon cluster including validator keys, charon p2p keys, and a cluster manifest. " +
+			"See flags for supported features.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runFunc(cmd.OutOrStdout(), conf)
+		},
+	}
+
+	bindClusterFlags(cmd.Flags(), &conf)
+
+	return cmd
+}
+
+// TODO(dhruv): replace newCreateClusterCmd with newCreateClusterCmdNew once charon-docker-compose are updated.
```

Comment:
can also add a comment before `newCreateClusterCmd` saying that it will be deprecated out

Reply:
actually `newCreateClusterCmdNew` will be renamed to `newCreateClusterCmd` and existing `newCreateClusterCmd` will be removed

Reply:
I agree @xenowits I agree the legacy commands should have a big "DEPRECATED" in they Short and Long fields. Independently of the name of the `new...Cmd` function

Reply:
Done!